lib/std/json.zig

```diff
@@ -310,6 +310,7 @@ pub const StreamingParser = struct {
         InvalidUtf8Byte,
         InvalidTopLevelTrailing,
         InvalidControlCharacter,
+        InsufficientSpace,
```

Comment:
Any new errors to `parseInternal` should be added to `ParseInternalErrorImpl` instead.